CHANGELOG.md

```diff
@@ -108,6 +108,7 @@
 * [ENHANCEMENT] Query-frontend: Include multiple tenant IDs in query logs when present instead of dropping them. #3125
 * [ENHANCEMENT] Alertmanager: reduced memory utilization in Mimir clusters with a large number of tenants. #3143
 * [ENHANCEMENT] Store-gateway: added extra span logging to improve observability. #3131
+* [ENHANCEMENT] Query-frontend: Log more detailed information in the case of a failed query. #3190
```

Comment:
Shouldn't this be in the "Grafana Mimir - main / unreleased" section?

Reply:
Yup, will fix